hostscripts/rpm-packaging/createproject.py

```diff
@@ -82,10 +82,10 @@ def upload_meta(project, build_repository, linkproject):
                   'projectlink': projectlink,
                   'build_repository': build_repository})
 
-    with tempfile.NamedTemporaryFile(delete=False) as meta:
+    with tempfile.NamedTemporaryFile() as meta:
         meta.write(templ)
+        meta.file.flush()
```

Comment:
why meta.file.flush() rather than meta.flush() ? the documentation says NamedTemporaryFile() is a file-like object..